src/ncp/ncp_openthread.cpp

```diff
@@ -177,6 +178,10 @@ void ControllerOpenThread::Init(void)
 
     mThreadHelper = std::unique_ptr<otbr::agent::ThreadHelper>(new otbr::agent::ThreadHelper(mInstance, this));
 
+#if OTBR_ENABLE_NAT64
+    otNat64SetEnabled(mInstance, true);
+#endif
```

Comment:
nit: move the block right next to the otSrpServerSetEnabled block since both are making enablement